packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor.light.scss

```diff
@@ -0,0 +1,34 @@
+.prosemirror pre {
+  white-space: pre-wrap;
+}
+
```

Comment:
Is there a better class to hang the styles off?
If you use another library you won't get `.prosemirror `or `.tiptap`.

Reply:
Good spot, I am now just applying an `editor-styles` class to the editor on load. I assume other editors might have the same functionality where they can add a class.